### DISCUSSION TITLE
Hydrogen with Yamaha midi, hi-hat question

### DISCUSSION THREAD

**Discussion by linuxzenassistance** [hydrogen-music/hydrogen#1528] [General]
Hi,
I use Hydrogen to generate drum sounds from a Yamaha DTX582K battery (DTX4 module) connected to midi.
Everything is perfect except for one thing: when I play a splash with the hi-hat at my foot, the sound continues even when I then close the hi-hat.
I need a fairly long foot splash sound, eg for ballads, but I would like if I play the hi-hat with the sticks or with the foot it immediately stops the sound triggered by the kick.
I can't find in the mod how to do this.
Do I have a way to do this in Hydrogen?
Thanks in advance for the reply.
carl

**Comment by cme** [comment]
Hi! You should be able to do this by setting the [mute group](http://hydrogen-music.org/documentation/manual/manual_en.html#chpt.instrument_editor.gain_and_mute_group) of all the hi-hat instruments (open, close, pedal) to the same group, so that starting one sound will stop the others. 

I can't remember why this isn't set by default in the kit.

**Reply by linuxzenassistance** [reply]
Thanks for the answer. That resolve perfectly my problem :-)
